python/revng/model/migrations/migrate.py

```diff
@@ -0,0 +1,41 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+from revng.model import latest_version  # type: ignore
+
+from revng.model.migrations.versions.v2 import Migrate2
+from revng.model.migrations.versions.v3 import Migrate3
+from .migration import Migration
+
+migrations: dict[int, Migration] = {
+    2: Migrate2(),
+    3: Migrate3(),
+}
+
+
+def migrate(model: dict):
+    if "Version" not in model:
+        return
```

Comment:
Add a comment about this, why are we just returning?
We should probably log something to `stderr`.

Reply:
The current design assumes that

1. a model with no version
2. a model with version set to 0
3. a model with version set to the schema version of revng

are all equivalent. This is why we are returning here. Since a model with no version is not ill-formed, I don't think a message (whether its error or warning) is warranted here, but maybe you have a different view on things.

Reply:
Add a comment about your reasoning.